commune_app/migrations/0003_createsuperuser.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import unicode_literals
+from django.db import migrations, transaction
+from commune_app.all_models.users import User
+
+
+def create_superuser(apps, schema_editor):
+    with transaction.atomic():
+        superuser = User()
+        superuser.is_active = True
+        superuser.is_superuser = True
+        superuser.is_staff = True
+        superuser.username = 'admin'
+        superuser.email = 'admin@gmail.com'
+        superuser.set_password('adminpassword')
```

Comment:
Please try to initiate the user with most of the variables like:
```suggestion
        superuser = User(is_active=True, is_superuser=True, is_staff=True, username='admin', email='admin@gmail.com', password='adminpassword')
```